nodes/materials/emission.py

```diff
@@ -119,6 +119,8 @@ def export_emission(self, exporter, depsgraph, props, definitions):
         if self.use_advanced:
             definitions["emission.power"] = self.power
             definitions["emission.efficency"] = self.efficacy
+            if self.power == 0 or self.efficacy == 0:
+                definitions["emission.gain"] = 0
```

Comment:
gain has to be a list with 3 elements I think, otherwise LuxCore will throw an error.

Reply:
Yes, that is correct. In the first lines a 3 tuple is used. I will push a fix.